File: dbflow/src/main/java/com/raizlabs/android/dbflow/structure/InvalidDBConfiguration.java
package com.raizlabs.android.dbflow.structure;

/**
 * Description: Thrown when a DB is incorrectly configured.
 */
public class InvalidDBConfiguration extends RuntimeException {

    public InvalidDBConfiguration() {
        super("No Databases were found. Did you create a Database Annotation placeholder class?");
    }

    public InvalidDBConfiguration(String message) {
        super(message);
    }
}

File: dbflow/src/main/java/com/raizlabs/android/dbflow/structure/InvalidDBConfiguration.kt
package com.raizlabs.android.dbflow.structure

/**
 * Description: Thrown when a DB is incorrectly configured.
 */
class InvalidDBConfiguration : RuntimeException {

    constructor() : super("No Databases were found. Did you create a Database Annotation placeholder class?")

    constructor(message: String) : super(message) {}
}
